### ISSUE TITLE
[BUG] `node_drain` spec test is being skipped on Github Actions

### ISSUE THREAD

**Issue by HashNuke** [lfn-cnti/testsuite#1435]
## Describe the bug

The spec test for `node_drain` is being skipped on the CI due to the condition specified in the spec test.

<img width="910" alt="CleanShot 2022-05-05 at 07 51 50@2x" src="https://user-images.githubusercontent.com/84005/166855262-8df86c20-4668-4a40-8339-8cd6dbf8227d.png">

## Proposed changes
Create a two-node kind cluster on GitHub Actions for the `node_drain` spec test to be run. This will help ensure that the pass scenario is always tested.

**Comment by lixuna** [comment]
@HashNuke  is still issue still relevant for CNTI?

**Comment by HashNuke** [comment]
@lixuna Yes this is still valid. I just checked one of the recent builds to confirm that this is still an issue.

**Comment by kosstennbl** [comment]
Still valid. 
![image](https://github.com/user-attachments/assets/97166fe8-0150-41c9-85ce-ecb10626943f)

**Comment by martin-mat** [comment]
node_drain being executed in gh actions
